sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -123,38 +136,32 @@ public ReactorConnection(String connectionId, ConnectionOptions connectionOption
                 return activeEndpoint.thenReturn(reactorConnection);
             })
             .doOnError(error -> {
-                final String message = String.format(
-                    "connectionId[%s] Error occurred while connection was starting. Error: %s", connectionId, error);
-
                 if (isDisposed.getAndSet(true)) {
-                    logger.verbose("connectionId[{}] was already disposed. {}", connectionId, message);
+                    logger.verbose("Connection was already disposed: Error occurred while connection was starting.", error);
                 } else {
-                    closeAsync(new AmqpShutdownSignal(false, false, message)).subscribe();
+                    closeAsync(new AmqpShutdownSignal(false, false, String.format(
+                        "Error occurred while connection was starting. Error: %s", error))).subscribe();
```

Comment:
Can the String format use `+` concatenation instead of `String.format` which requires a parsing of the string